env0/resource_environment_test.go

```diff
@@ -955,9 +955,18 @@ func TestUnitTemplatelessEnvironmentResource(t *testing.T) {
 	resourceName := "test"
 	accessor := resourceAccessor(resourceType, resourceName)
 
+	environment := client.Environment{
```

Comment:
acceptance test has been updated.
Before it just tested the schema (plan only).
Now it tests Create and Read.